app/models/user.py

```diff
@@ -352,3 +355,19 @@ def __setattr__(self, name, value):
 @event.listens_for(User, 'init')
 def receive_init(target, args, kwargs):
     target.signup_at = datetime.now(pytz.utc)
+
+
+@event.listens_for(User, 'after_update')
```

Comment:
@niranjan94 This way we don't need to rely on API resource to send the email. If the email is updated user model will send the email.